bin/test

```diff
@@ -58,6 +29,81 @@ var testsWorkspace = process.env.CODE_TESTS_WORKSPACE || testsFolder;
 console.log('### VS Code Extension Test Run ###');
 console.log('Current working directory: ' + process.cwd());
 
+function getTags(url, tags) {
+    var tags = tags || [];
+
+    return ghGot(url || 'repos/Microsoft/vscode/tags')
+        .then(function(result) {
+            var link = parse(result.headers.link);
+            tags = tags.concat(result.body);
+
+            // Recursively load the tags of all the pages
+            if (link.next) {
+                return getTags(link.next, tags);
+            }
+
+            return tags;
+        });
+}
+
+function getDownloadUrl() {
+    if (process.env.CODE_DOWNLOAD_URL) {
+        return Promise.resolve(process.env.CODE_DOWNLOAD_URL);
+    }
+
+    return getTags()
+        .then(function(tags) {
+            // Filter out invalid tags and prerelease tags
+            tags = tags.filter(function(tag) {
+                var parsed = semver.parse(tag.name);
+                return parsed && parsed.prerelease.length === 0;
+            });
```

Comment:
It's correct that I can't install an insiders release right? I tried it with `0.10.7-insiders`, but it didn't work so that's why I'm filtering them here.